samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/SubmitToGradingActionListener.java

```diff
@@ -979,16 +980,26 @@ private void handleMarkForReview(List<ItemGradingData> grading, HashSet<ItemGrad
     }
 
    
-	private Boolean isLate(PublishedAssessmentIfc pub, boolean submitFromTimeoutPopup) {
+	private Boolean isLate(PublishedAssessmentIfc pub, boolean submitFromTimeoutPopup, String adataAgentId) {
 		AssessmentAccessControlIfc a = pub.getAssessmentAccessControl();
 		// If submit from timeout popup, we don't record LATE
 		if(submitFromTimeoutPopup) {
 			return Boolean.FALSE;
-		}		
-		
+		}
+
+		Boolean isLate = false;
 		if (a.getDueDate() != null && a.getDueDate().before(new Date()))
-			return Boolean.TRUE;
+			isLate = Boolean.TRUE;
 		else
-			return Boolean.FALSE;
+			isLate = Boolean.FALSE;
```

Comment:
Can you add brackets around the if/else?

Reply:
Done